research/delf/delf/python/training/README.md

```diff
@@ -164,6 +164,22 @@ the batch size to `256`:
 --batch_size=256
 ```
 
+It is also possible to train the model with an improved global features head as
+introduced in the [DELG paper](https://arxiv.org/abs/2001.05027). To do this,
+specify the additional parameter `--delg_global_features` when launching the 
+training, like in the following example:
+
+```
+python3 train.py \
+  --train_file_pattern=gldv2_dataset/tfrecord/train* \
+  --validation_file_pattern=gldv2_dataset/tfrecord/validation* \
+  --imagenet_checkpoint=resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5 \
+  --dataset_version=gld_v2_clean \
+  --logdir=gldv2_training/ \
+  --delg_global_features
+```
+
+
```

Comment:
quick sanity check, will the model trained in this manner work with the export commands below? Maybe there needs to be some edit? (eg, I think the global exporting script would need some arguments to denote that new pooling/embedding layers would be used.)

Reply:
Thanks for the feedback André! I will check for backwards compatibility and update the PR accordingly.

Reply:
I have successfully tested 1) exporting the DELF local feature model with the newly trained model, 2) extracting the DELF features for 2 test images and 3) matching based on the extracted features and it works as before. Also extracting the Kaggle-compatible global feature model works with the newly trained model, however the script instantiates a Delf() model not a Delg() model. There are indeed changes to be made to the global feature model script (examples/export_global_model.py) in order to allow for exporting a Delg() model. These changes being Python code I would need to make a new commit. Please let me know if you want me to mix the Python changes with this README update in the same PR or if I should make a separate PR with the Python code only (and potentially some additional README update). Thank you!

Reply:
Thanks Dan! Yes, I agree we can do those changes in a separate commit. Let me merge this PR now.